doc/src/docs/InputOutputReference/01c-InputOutputReference.md

```diff
@@ -15446,6 +15448,183 @@ This field reports the amount of radiant energy transferred from the baseboard t
+
+#### Field: Rated Capacity
+
+This field is the rated radiant cooling panel capacity in Watts at a rated water mass flow rate (see previous field).  Most manufacturers will publish a capacity in their literature.  This field can be autosized by EnergyPlus. If it is blank or zero, autosizing is assumed.
+
+#### Field: Maximum Chilled Water Flow Rate
+
+The maximum chilled water volumetric flow rate in m3/sec.  This field can also be autosized.
+
+#### Field: Control Type
+
+This field specifies two different types of control categories: temperature or zone load.  For one of the temperature based controls, this field along with the throttling range and setpoint schedules determines how the user wishes to control the radiant cooling panel.  The temperature denoted in the setpoint schedule can refer to one of five different temperatures: the zone mean air temperature, the zone mean radiant temperature, the zone operative temperature, the outdoor dry-bulb temperature, or the outdoor wet-bulb temperature.  The current field controls the choice of temperature.  For controls that are based more on zone load, the user can make one of two choices based on whether the zone load is met by only the convective output of the unit or the total output of the unit.  These zone load controls will not perform as precisely as air-based systems.  The convective zone load control will come close to meeting the current setpoint temperature (defined by the typical zone thermostat input by the user) since the convective output of the unit will be set to match the remaining zone load.  However, in the long run, this will likely overcool the space since radiant energy is being absorbed by surfaces and this will impact the zone conditions at later time steps.  The total zone load control will likely undercool the space in the short term since the total output (radiant and convective) of the unit will be set to meet the remaining zone load.  However, this will not fully impact the zone air temperature immediately since some of the energy is being absorbed from surfaces rather than the zone air.  The user must select from the following options:
+
* MeanAirTemperature
+
* MeanRadiantTemperature

* OperativeTemperature

* OutdoorDryBulbTemperature

* OutdoorWetBulbTemperature

* ZoneConvectiveLoad

* ZoneTotalLoad

Operative temperature for radiant system controls is the average of Mean Air Temperature and Mean Radiant Temperature. If the user does not select a control type, MeanAirTemperature control is assumed by EnergyPlus. See the throttling range and control temperature schedule fields below for more information.
+
```

Comment:
Something looks super wonky with the bulleted list above.  Git thinks they are all on the same line??  Maybe a line ending problem?